unified-installer/astra-unified-installer.sh

```diff
@@ -65,6 +68,9 @@ readonly __COMPONENTS_ACP_ONLY="ACP_ONLY"
 readonly __COMPONENTS_VALID_VALUES=("$__COMPONENTS_ALL_ASTRA_CONTROL" "$__COMPONENTS_TRIDENT_AND_ACP" \
     "$__COMPONENTS_TRIDENT_ONLY" "$__COMPONENTS_ACP_ONLY")
 
+readonly __ASTRA_API_TOKEN_PULL_SECRET="astra-api-token"
+readonly __ASTRA_REGISTRY_IMAGE_PULL_SECRET="astra-regcred"
```

Comment:
Just making the names official since it was hardcoded in a few different places.